src/screens/SendSheet.js

```diff
@@ -157,17 +156,26 @@ export default function SendSheet(props) {
   const showAssetForm = isValidAddress && !isEmpty(selected);
 
   const isNft = selected?.type === AssetTypes.nft;
-  let color = useColorForAsset({
-    address: selected?.mainnet_address || selected.address,
-  });
+  let colorForAsset = useColorForAsset(
+    {
+      address: selected?.mainnet_address || selected.address,
+    },
+    null,
+    false,
+    true
+  );
   if (isNft) {
-    color = colors.appleBlue;
+    colorForAsset = colors.appleBlue;
   }
 
   const isL2 = useMemo(() => {
     return isL2Network(currentNetwork);
   }, [currentNetwork]);
 
+  const transactionNetwork = useMemo(() => {
```

Comment:
why are we getting `transactionNetwork` separately? Shouldn't the useEffect on line 327 (with the network specific `setCurrentNetwork`) handle this for us?

Reply:
very true